.travis.yml

```diff
@@ -48,6 +48,10 @@ matrix:
       env: COMPANION=true
       addons:
         apt: *APT
+    - node_js: 6
+      env: COMPANION_NODE6=true
```

Comment:
We can just reuse the `COMPANION=true` env var here, I think, similarly to how we have 2 jobs for `UNIT=true` (one on stable node and one on node 10)

Reply:
yes but it'll run deployment 2ce I think. I added `COMPANION_NODE6` to avoid that

Reply:
ah yes. all good then :+1: